### PR TITLE
LibCore+Base: Update man pages for utilities

### PR DESCRIPTION
**Pull request by tcl3**
This PR updates the existing man pages for user facing command line programs. These man pages should now document all currently available options. I have also made sure the formatting is consistent between man pages.

### CODE REVIEW

File: Base/usr/share/man/man1/tar.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Files may also be compressed and decompressed using GNU Zip (GZIP) compression.
 * `-t`, `--list`: List contents
 * `-v`, `--verbose`: Print paths
 * `-z`, `--gzip`: Compress or decompress file using gzip
+* `-lzma`: Compress or decompress file using lzma
```
Comment (AtkinsSJ) [inline Base/usr/share/man/man1/tar.md:25]: This should be `--lzma`

**Review by AtkinsSJ — CHANGES_REQUESTED** [review]
I spotted one typo, but this is very helpful! Thanks for going through all those. :^)

File: Base/usr/share/man/man1/tar.md (hunk 1 above)
Comment (tcl3) [reply Base/usr/share/man/man1/tar.md]: Oops! Well spotted :^)

**Comment by tcl3** [comment]
Latest push removes colons from generated markdown header names, for better consistency.

**Review by AtkinsSJ — APPROVED** [review]
👍

**Pull request merged**